.github/workflows/external-data-checker.yml

```diff
@@ -0,0 +1,12 @@
+name: 'Check if external data changed across Flathub org'
```

Comment:
This and other leftovers from the master branch should be removed. Other than that, LGTM.